src/new-client/src/App/Trades/TradesHeader/TradesHeader.tsx

```diff
@@ -30,17 +47,49 @@ const HeaderToolbar = styled("div")`
   align-items: center;
   justify-content: flex-end;
 `
-
-export const TradesHeader: React.FC = () => (
-  <TradesHeaderStyle>
-    <HeaderLeftGroup>Trades</HeaderLeftGroup>
-    <HeaderRightGroup>
-      <ExcelButton />
-      <HeaderToolbar>
-        <AppliedFilters />
-        <QuickFilter />
-      </HeaderToolbar>
-      <Fill />
-    </HeaderRightGroup>
-  </TradesHeaderStyle>
-)
+var isTornOut = false
+export const TradesHeader: React.FC = () => {
+  const tearOutTileState = useTearOutTileState("Tiles")
+  const tearOutAnalyticsState = useTearOutAnalyticsState("Analytics")
+  useObservableSubscription(
+    tearOutEntryTrades$.subscribe(async ([tornOut]) => {
+      if (tornOut && !isTornOut) {
+        isTornOut = true
+        openWindow(
```

Comment:
In common App components we shouldn't use any platform specific stuff (web, openfin ec..)
Here you should use `openWindow` from `utils/window`.